### PR TITLE
Template and in-memory sample for Freyja apps

### PR DESCRIPTION
**Pull request by wilyle**
Adds a template and in-memory sample for Freyja apps. Also includes some initial docs and workflows for the repo

### CODE REVIEW

File: freyja_apps/in_memory/README.md
Comment (ashbeitz) [inline freyja_apps/in_memory/README.md:10]: You can condense this down to just the run command.

File: freyja_apps/in_memory/README.md
Comment (ashbeitz) [inline freyja_apps/in_memory/README.md:14]: replace the "build" with "run"

File: freyja_apps/template/README.md
Comment (ashbeitz) [inline freyja_apps/template/README.md:10]: Please place periods at the end of each of the numbered lines.

File: freyja_apps/template/README.md
Comment (ashbeitz) [inline freyja_apps/template/README.md:22]: No need for cargo build.

**Review by ashbeitz — COMMENTED** [review]
Please take a look at my comments.

**Review by ashbeitz — APPROVED** [review]
Approved

**Review by jorchiu — APPROVED** [review]

File: freyja_apps/template/Cargo.template.toml
Comment (ladatz) [inline freyja_apps/template/Cargo.template.toml:12]: Similar to my other comment: consider adding a comment about how to choose the commit hash here

Comment (ladatz) [reply freyja_apps/template/Cargo.template.toml:12]: Ah, now I see it's in the documentation. Up to you if you want to add it here too

**Review by ladatz — APPROVED** [review]
Approving with just one suggestion

File: freyja_apps/template/Cargo.template.toml
Comment (wilyle) [reply freyja_apps/template/Cargo.template.toml:12]: I'll leave it to the docs

**Pull request merged**
